src/OpenLoco/CMakeLists.txt

```diff
@@ -644,6 +640,13 @@ if (APPLE AND FALSE) # TODO: This is broken after moving resources into src/Reso
     )
 endif ()
 
+find_package(yaml-cpp)
+find_package(span-lite)
```

Comment:
You don't need to have this here as well as in the thirdparty file. Can be removed.

Reply:
I'm battling with the linker on span-lite also, both when I leave the non-std::span-lite or change it to simply span-lite.
```
CMake Error at src/OpenLoco/CMakeLists.txt:544 (target_link_libraries):
  Target "OpenLoco" links to:

    nonstd::span-lite

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
```